source/compatibility.tex

```diff
@@ -240,6 +240,18 @@
 \tcode{trivially_relocatable_if_eligible} as macros is invalid
 in this revision of \Cpp{}.
 
+\rSec2[diff.cpp23.mem]{\ref{mem}: memory management library}
```

Comment:
The paper has "Memory" with uppercase M but it seems that is wrong and it should indeed be lowercase. But "C.1.2 Clause 5: Lexical conventions" is wrong in the current WP.

Reply:
I'll fix that, thanks!